docs/nomenclature.md

```diff
@@ -1,13 +1,22 @@
 Nomenclature
 ============
 
-The following explains several nouns you may experience in mutant's documentation.
+This document explains several nouns you may experience in mutant's documentation.
 It's a good idea to familiarize yourself before moving on.
 
+## Mutation Testing
+The practice of systematically applying small changes one at a time to a codebase then re-running the (relevant) tests for each change. If the tests fail, then they cover the semantics of those changes. If the tests continue to pass with the changes in place, the tests do not cover the complete semantics of the changed parts of the codebase.
+
+Each type of change (for example, “change `a && b` to `a`”) is known as a [Mutation Operator](#mutation-operator), and is applied to the pristine codebase - that is, the changes are not stacked up, but applied in isolation. The places in which the Mutation Operators can make their changes are called [Subjects](#subject). The changed Subject is then referred to as a [Mutation](#mutation).
```

Comment:
"Each type of change (for example, “change `a && b` to `a`”) is known as a ..."

This sounds like the change itself is a "Mutation Operator" but I think that would be a [Mutation](https://github.com/mbj/mutant/pull/1146/files#diff-30f1590c9d25c9533cc089fcdfed706a2b237024223e18ca16db6d857b2de656R80-R81). You mean to be talking about kinds of changes but I think the example confuses it a bit because it is an instance of a mutation.